packages/evm/src/precompiles/distribution.ts

```diff
@@ -112,7 +119,14 @@ export const DISTRIBUTION_PRECOMPILE_ABI: Abi = [
 		outputs: [{ internalType: 'bool', name: 'success', type: 'bool' }],
 		stateMutability: 'nonpayable',
 		type: 'function'
-	}
+	},
+  {
```

Comment:
Just curious, does this have to match the abi in the sei-chain repo?
https://github.com/sei-protocol/sei-chain/blob/main/precompiles/distribution/abi.json

Not seeing it there in `main` (but also not sure if that's the right branch rn)

Reply:
Yup, does have to match. There is a separate PR for that https://github.com/sei-protocol/sei-chain/pull/1670